build_rules.gradle

```diff
@@ -489,6 +489,22 @@ ext.applyDockerNature = {
   }
 }
 
+def containerImageName(name) {
```

Comment:
Add method comment.

Interesting that def works, before I was sticking everything onto ext. This seems much cleaner. Consider swapping all usages of ext.fooBar = {} to def fooBar

Reply:
Added. I also stick it into ext below.